docker-compose.yml

```diff
@@ -47,6 +47,7 @@ services:
       - AWS_EXECUTION_ENV=True
       - DEFAULT_REGION=eu-west-2
       - HOSTNAME_EXTERNAL=${HOSTNAME_EXTERNAL-localstack}
+      - SQS_PROVIDER=elasticmq
```

Comment:
I think this probably isn't needed any more?

Reply:
Yes, you are right.
I will remove this flag.